docs/pages/hdmi-cec-sink-L3-Low-Level_TestSpec.md

```diff
@@ -0,0 +1,346 @@
+
+- Reference to the tool: https://www.pulse-eight.com/p/104/usb-hdmi-cec-adapter
+- libcec tools: https://www.pulse-eight.com/Download/Get/51  
+
+**Libraries and tools required for RAFT**
+- libcec should be installed on the PC where the Pulse-Eight CEC Adaptor is connected.
+- python-cec will be used to control the `CEC` adaptor.
+- https://pypi.org/project/cec/ will provide more information on how to use this library.
+  
+**cec-ctrl Commands used in Manual Mode**
+- libcec should be installed on the PC where the Pulse-Eight CEC Adaptor is connected.
+- To get the logical and physical address: `cec-client -s -d 4`
+- To Tx data: `cec-client -s -t 1 --data <command>` or `echo "tx <data>" | cec-client -s` Eg:`echo "tx 10:36" | cec-client -s`
+- To check the last received data use `cec-client -m | grep '>>' | tail -n 1`.  `cec-client -m` shall monitor the data continuously 
+  
+**Prerequisite Test to make sure all the CEC Adaptors are connected**
+1. Make sure to read the physical address of the CEC Adaptor allocated based on the port it has been connected to, using libCEC on PC.
+2. The physical and logical addresses allocated to the CEC adaptor shall be read and validated on RAFT.
+
+## Test Functionalities
+
+#### Message Transmission and Reception Test
+Note: All the below tests should be carried out on all the available HMDI ports.
+- Transmit an HDMI CEC basic command (GetCECVersion) from the DUT to receive a reply from the connected CEC Adaptor.
+- Receive an HDMI OSD Command with a string of maximum length (14 bytes) from the CEC Adaptor and acknowledge it.
+- The CEC Adaptor sends a standby command to a device to all the logical addresses other than 0x00 and 0x0f.  `DUT` should not respond to any of these commands. 
+
+#### Message Broadcast and Receive broadcast 
+Note: All the below tests should be carried out on all the available HMDI ports.
+- Broadcast an HDMI CEC Command from the DUT and verify that this command has been received on the CEC Adaptor.
+- Receive a standby broadcasting command on the DUT sent by the CEC Adaptor and validate it.
+  
+#### Stress Test
+- Receiving an HDMI OSD Command repeatedly for 10 times with a different string of max length from CEC Adaptor.
+- Transmit an HDMI OSD Command repeatedly for 10 times with a different string of max length to the CEC Adaptor.
+
+#### Hardware Fault Test
+- Introducing a Fault on the HDMI line to test the Transmit functionality.
+
+# Test 1: Message Transmission and Reception Test - Unicast messages
+
+Functionality: 
+1. RAFT/user to send a simple "getCECVersion" command to the `DUT` before setting the logical address.
+2. `DUT` shall request a CEC Version from the CEC adaptor connected.  `DUT` should receive a valid CEC version.
+3. `DUT` shall receive an OSD Command with max buffer size from the CEC Adaptor and acknowledge to this command.
```

Comment:
this is all very confused on what's expected here and doesn't define the process for how this will work.

Target :Trigger the menu test on L3, Wait for CEC Response.
Host : Send the yaml message to the system.
Target: Will output the message sent
Host: Will capture the message and compare the result against the expected result embedded in the yaml.

Repeat as required, define the list of yaml messages with their content, and expected result.